### PR TITLE
feat: script to manage ownership

### PR DESCRIPTION
**Pull request by deanamiel**
Script to manage ownership for all contracts that inherit the SDK Ownable.sol as per AXE-1594:
https://axelarnetwork.atlassian.net/browse/AXE-1594

### CODE REVIEW

File: evm/ownership.js [hunk 1]
```diff
@@ -0,0 +1,200 @@
...
+async function main(options) {
+    const config = loadConfig(options.env);
+
+    const chain = options.chain;
+
+    if (config.chains[chain.toLowerCase()] === undefined) {
+        throw new Error(`Chain ${chain} is not defined in the info file`);
+    }
+
+    await processCommand(options, config.chains[chain.toLowerCase()], config);
```
Comment (blockchainguyy) [inline evm/ownership.js:165]: Function definition has 2 input args: `options`, `chain`. here we are supplying 3 args. We should remove `config` if not used.

**Comment by blockchainguyy** [comment]
https://github.com/axelarnetwork/axelar-gmp-sdk-solidity/blob/main/contracts/interfaces/IOwnable.sol

This interface implements both:
- transfer ownership in a single tx
- 2 step transfer: Propose and accept

Why have we implemented both options?

File: evm/ownership.js [hunk 2]
```diff
@@ -0,0 +1,200 @@
...
+
+    const rpc = chain.rpc;
+    const provider = getDefaultProvider(rpc);
+
+    const wallet = new Wallet(privateKey, provider);
+    await printWalletInfo(wallet);
+
+    printInfo('Contract name', contractName);
+
+    const contractPath = artifactPath + contractName + '.sol/' + contractName + '.json';
```
[the remark this answers is not in the document]
Comment (deanamiel) [reply evm/ownership.js:34]: changed to use IOwnable and added an optional address override

File: evm/ownership.js (hunk 1 above)
Comment (deanamiel) [reply evm/ownership.js:165]: Good catch

**Comment by deanamiel** [comment]
> https://github.com/axelarnetwork/axelar-gmp-sdk-solidity/blob/main/contracts/interfaces/IOwnable.sol
> 
> This interface implements both:
> 
> * transfer ownership in a single tx
> * 2 step transfer: Propose and accept
> 
> Why have we implemented both options?

My understanding is that it provides more flexibility. There may be some circumstances where we want to perform a single step ownership transfer, but other instances where it would be more secure to perform a 2-step transfer where the proposed owner has to accept the ownership role. Not only for us, but also for developers using our SDK.

**Review by milapsheth — APPROVED** [review]

**Pull request merged**